### DISCUSSION TITLE
SideDownload Error

### DISCUSSION THREAD

**Discussion by shaboingery22** [aclap-dev/video-downloadhelper#525] [Help and Q&A]
### Your question

hello, new user attempting from this url https://www5.javmost.com/CJOD-280-UNCENSORED-LEAK/
facing this error below. any help would be appreciated. DL seems to time out at ~20%

bs: SideDownload error: 
    at Object.bT (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:44:987)
    at async RT (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:44:11323)
    at async ri (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:44:13259)

### Addon & CoApp version and details
version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-GB
license: unset
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/Users/inky",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "n6.1-11-gc30f360955"
}

**Comment by GrampaWildWilly** [comment]
You must follow the instructions on the page where you created your problem report.  You skipped some of them.  They are clearly documented.  Under the heading "Addon & CoApp version and details" you need to supply the versions of your software, not the error message you got.  There are instructions for that section of the problem report on the problem creation page.  You need to read them & follow them.  But you don't need to create a new problem to find those instructions.  There is a link labelled Wiki at the top of this page.  Click that.  You will get what amounts to a user manual.  Read that.  You'll find the instructions you skipped.  Fix your problem report.  Notice the 3 dots above & to the right of your problem report.  Click that.  I think you'll be able to take it from there.

**Comment by GrampaWildWilly** [comment]
Good job!

I do not work for VDH.  I'm just another user.

I visited that web page & VDH was able to download an MP4 of size 2.31G, duration 1:58:40, in 9:27 without a problem.  But I am running Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.13a1 beta, CoApp 2.0.19.  I know you are running Chrome, but there is a beta version of VDH for Chrome that is comparable to the one I am using on Firefox.  You should try that.

Also, you should read #519.  Be sure you click any links that say, "Show x previous replies."  The ideas discussed there might apply to your web site as well.
